### ISSUE TITLE
ipc for 0.9.91 worked and now 0.9.92 and 0.9.93 appears to have a problem

### ISSUE THREAD

**Issue by jeremyf** [Hammerspoon/hammerspoon#3024]
In 0.9.91, I had a `hs` command.  When I upgraded to 0.9.92 and 0.9.93 that's gone.

In my Console I have the following:

```
2021-12-13 22:53:40: Welcome to the Hammerspoon Console!
You can run any Lua code in here.

2021-12-13 22:53:40: -- Lazy extension loading enabled
2021-12-13 22:53:40: -- Loading ~/.hammerspoon/init.lua
2021-12-13 22:53:40: -- Loading Spoon: editWithEmacs
2021-12-13 22:53:40: Starting loading editWithEmacs spoon
2021-12-13 22:53:40: -- Loading extension: ipc
2021-12-13 22:53:40: cli installation problem: 'hs' is a dangling link.
2021-12-13 22:53:40: cli installation problem: man page for 'hs.1' is a dangling link.
```

These new versions broke the command line interactions.

**Comment by jeremyf** [comment]
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Aliquam id diam maecenas ultricies mi eget mauris. Rutrum quisque non tellus orci ac auctor augue mauris augue. Id ornare arcu odio ut sem nulla pharetra diam. Viverra justo nec ultrices dui sapien eget. Sit amet nisl suscipit adipiscing bibendum est. Vitae turpis massa sed elementum tempus. Sed elementum tempus egestas sed. Tempor orci eu lobortis elementum nibh tellus. At volutpat diam ut venenatis.

**Comment by luckman212** [comment]
I had the same thing. Try running the following from your HS console:

```lua
hs.ipc.cliUninstall()
hs.ipc.cliInstall()
```

**Comment by cmsj** [comment]
0.9.92 reorganised the `Hammerspoon.app` bundle to be more correct (we had executable code libraries in the wrong place) and that meant moving the precise location of the `hs` binary within the bundle, making your existing symlinks point at a non-existent file. Thanks @luckman212 for helping out.

**Comment by rsefer** [comment]
I have this same issue and the re-install does not work (returns false). I had this issue before when I switched to the M1 Pro / Apple Silicon and unfortunately do not remember how I fixed it pre 0.9.92

**Comment by jeremyf** [comment]
@luckman212 I'm on M1 Big Sur and your guidance helped get things working again!

Thank you all for your help and Hammerspoon!
